doc/whats_new/v1.0.rst

```diff
@@ -320,6 +320,14 @@ Changelog
 - |Fix| Allow multiple scorers input to
   :func:`~sklearn.inspection.permutation_importance`.
   :pr:`19411` by :user:`Simona Maggio <simonamaggio>`.
+  
+:mod:`sklearn.kernel_approximation`
+.......................
```

Comment:
```suggestion
:mod:`sklearn.kernel_approximation`
...................................
```

Reply:
Thank you for your suggestions. The doc has been revised, and a non-regression test has been added.